aeppl/logprob.py

```diff
@@ -596,3 +596,10 @@ def multinomial_logprob(op, values, *inputs, **kwargs):
         at.all(at.ge(n, 0)),
     )
     return res
+
+
+@_logprob.register(DiracDelta)
+def diracdelta_logprob(op, values, *inputs, **kwargs):
+    (values,) = values
+    (const_value,) = inputs
+    return at.switch(at.eq(values, const_value), 0.0, -np.inf)
```

Comment:
Os strict equality or `isclose` best here?

Reply:
Yeah, we could use that.